bin/context-adapter

```diff
@@ -27,4 +27,4 @@
 
 var contextAdapter = require('../lib/context_adapter');
 
-contextAdapter.startup();
+contextAdapter.start(true);
```

Comment:
Fixed in https://github.com/telefonicaid/context-adapter/commit/7fb519395ac79dd516ae4d8fbbbf4dc179870283